modules/installation/proc-install-rhdh-helm.adoc

```diff
@@ -21,10 +21,19 @@ The {product} Helm chart is available in the Helm catalog in Red Hat OpenShift D
 
 . Switch to *Developer* mode on your Red Hat OpenShift web console.
```

Comment:
Per a recent discussion, consider whether we want to align our docs to other CCS docs by using the term _Developer perspective_ in place of _Developer mode_

Reply:
Done